Cubical/Foundations/HLevels.agda

```diff
@@ -433,9 +434,13 @@ isPropΠ4 : (h : (x : A) (y : B x) (z : C x y) (w : D x y z) → isProp (E x y z
 isPropΠ4 h = isPropΠ λ _ → isPropΠ3 (h _)
 
 isPropΠ5 : (h : (x : A) (y : B x) (z : C x y) (w : D x y z) (v : E x y z w) → isProp (F x y z w v))
-            → isProp ((x : A) (y : B x) (z : C x y) (w : D x y z)  (v : E x y z w) → F x y z w v)
+            → isProp ((x : A) (y : B x) (z : C x y) (w : D x y z) (v : E x y z w) → F x y z w v)
 isPropΠ5 h = isPropΠ λ _ → isPropΠ4 (h _)
 
+isPropΠ6 : (h : (x : A) (y : B x) (z : C x y) (w : D x y z) (v : E x y z w) (u : F x y z w v) → isProp (G x y z w v u))
+            → isProp ((x : A) (y : B x) (z : C x y) (w : D x y z) (v : E x y z w) (u : F x y z w v) → G x y z w v u)
+isPropΠ6 h = isPropΠ λ _ → isPropΠ5 (h _)
```

Comment:
Haha, this is starting to get crazy... Maybe it's time to think about an `isProp` solver...

Reply:
Couldn't agree more!